commands/pong.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 	name: "pong",
 	description: "Pong.",
 	botPermissions: ["EMBED_LINKS"],
-	async execute(message, args) {
+	async execute(message, args) { //removed args for testing - seems to of worked /shrug
```

Comment:
another comment here

Reply:
my random things to remember what I was testing. merging dev into main was a bit of a mistake lol